README-Automation.md

```diff
@@ -0,0 +1,195 @@
+#Automation Scripts
+
+## Table of Contents
+
+- [Services](#services)
+- [Installing The Automation Scripts](#installing-the-automation-scripts)
+  - [Stable Channel](#stable-channel)
+  - [Unstable Channel](#unstable-channel)
+- [Initializing Drive](#initializing-drive)
+- [Using The Automation Scripts](#using-the-automation-scripts)
+  - [Setting Up](#setting-up)
+  - [Selecting Objects](#selecting-objects)
+  - [Launching A Script](#launching-a-script)
+  - [Help](#help)
+- [drive-menu Screenshots](#drive-menu-screenshots)
+- [Tracking And Filing Issues](#tracking-and-filing-issues)
+- [Reaching Out](#reaching-out)
+- [Disclaimer](#disclaimer)
+- [LICENSE](#license)
+
+## Services
+
+JC Manciot's [PPA](https://gitlab.com/jean-christophe-manciot/ppa) offers a collection of scripts for automated drive commands including syncing which can be used on the latest stable Ubuntu distribution.
```

Comment:
> latest stable Ubuntu distribution

Which one? I noticed `yakkety` reference only (below).

Is it compatible with the latest LTS (`xenial`)? If no, I'd suggest highlighting this incompatibility (or if it hasn't been tested with Xenial - declare this; as many Ubuntu users prefer LTS rather than non-LTS).